libs/localization/src/lib/en.json

```diff
@@ -1850,7 +1850,7 @@
     "forecastHeader": "Forecast analysis",
     "whatIfForecastingDescription": "What-if allows you to perturb features for your entire time series and observe how the model's forecast changes.",
     "whatIfForecastingChooseTimeSeries": "To start, choose a time series from the options below.",
-    "forecastDescription": "Forecast analysis compares your model's forecast to the actual values of your time series.",
+    "forecastDescription": "Forecast analysis compares your model's forecast to the actual values of your time series. To enable what-if analysis, please provide a dataset with features.",
```

Comment:
v.nit: it is recommended to not use please in user facing statements. I know we have this in various places :(